tests/integration/conftest.py

```diff
@@ -1109,7 +1172,7 @@ def global_props(self):
         """The global properties for the test suite.
 
         Returns:
-            dict(str): Global properties for the test suite.
+            {str: obj}: Global properties for the test suite.
```

Comment:
The JetBrains IDE does not recognize "dict(*)" as a valid docstring data type. Fixed.